builtin/providers/aws/resource_aws_elb.go

```diff
@@ -121,6 +121,35 @@ func resourceAwsElb() *schema.Resource {
 				Default:  300,
 			},
 
+			"access_logs": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+							Default:  false,
+						},
+						"interval": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  60,
+						},
+						"bucket": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+							Default:  "",
+						},
+						"bucket_prefix": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "",
```

Comment:
I wasn't very clear in my first post, sorry :frowning: 

If we have a default of `""`, then Terraform will send `""` as the prefix in the struct. Assuming AWS doesn't just flat out reject that, you'll end up with this:

    mybucket//AWSLogs/123455

But I suspect it will plain reject it. 

The aws-sdk expects `nil` for "non existent" attributes, which is why the attributes are all pointers:

```go
// Information about the AccessLog attribute.
type AccessLog struct {
        [...]
	// The name of the Amazon S3 bucket where the access logs are stored.
	S3BucketName *string `type:"string"`

	// The logical hierarchy you created for your Amazon S3 bucket, for example
	// my-bucket-prefix/prod. If the prefix is not provided, the log is placed at
	// the root level of the bucket.
	S3BucketPrefix *string `type:"string"`
        [...]
}
```

I was poorly articulating that for `prefix`, we need to do the same as I [mentioned below][1], and  only add it if it's not `""`


[1]: https://github.com/hashicorp/terraform/pull/3062#discussion_r37881196